packages/next/client/link.tsx

```diff
@@ -97,13 +97,14 @@ function prefetch(
   prefetched[href + '%' + as] = true
 }
 
-function isNewTabRequest(event: React.MouseEvent) {
+function isModifiedEvent(event: React.MouseEvent) {
   const { target } = event.currentTarget as HTMLAnchorElement
   return (
     (target && target !== '_self') ||
     event.metaKey ||
     event.ctrlKey ||
     event.shiftKey ||
+    event.altKey ||
```

Comment:
Can we add a comment explaining this triggers resource download?

Reply:
I added.